hack/ci/run-tests-default-application-catalog.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### This script sets up a local KKP installation in kind and then
+### runs the conformance-tester to create userclusters and check their
+### Kubernetes conformance.
+
+set -euo pipefail
+
+cd $(dirname $0)/../..
+source hack/lib.sh
+
+export GIT_HEAD_HASH="$(git rev-parse HEAD)"
+export KUBERMATIC_VERSION="${GIT_HEAD_HASH}"
+
+TEST_NAME="Pre-warm Go build cache"
+echodate "Attempting to pre-warm Go build cache"
+
+beforeGocache=$(nowms)
+make download-gocache
+pushElapsed gocache_download_duration_milliseconds $beforeGocache
+
+echodate "Creating kind cluster"
+export KIND_CLUSTER_NAME="${SEED_NAME:-kubermatic}"
+source hack/ci/setup-kind-cluster.sh
+
+# gather the logs of all things in the cluster control plane and in the Kubermatic namespace
+protokol --kubeconfig "$KUBECONFIG" --flat --output "$ARTIFACTS/logs/cluster-control-plane" --namespace 'cluster-*' > /dev/null 2>&1 &
+protokol --kubeconfig "$KUBECONFIG" --flat --output "$ARTIFACTS/logs/kubermatic" --namespace kubermatic > /dev/null 2>&1 &
+
+echodate "Setting up Kubermatic in kind on revision ${KUBERMATIC_VERSION}"
+
+export KUBERMATIC_YAML=hack/ci/testdata/kubermatic.yaml
+
+beforeKubermaticSetup=$(nowms)
+kubectl apply -f pkg/crd/k8c.io
+source hack/ci/setup-kubermatic-in-kind-and-test-default-application-catalog-apps.sh
```

Comment:
I would try to re-use `hack/ci/setup-kubermatic-in-kind.sh` here and run the additional steps you added to the script to _this_ script (`hack/ci/run-tests-default-application-catalog.sh`).